.github/workflows/build.yaml

```diff
@@ -38,6 +41,17 @@ jobs:
       - name: Code checkout
         uses: actions/checkout@v4
 
+      - name: Install dependencies
+        run: |
+          set -euxo pipefail
+
+          wget https://apt.llvm.org/llvm.sh -O /tmp/llvm.sh
```

Comment:
I think that there is another way to solve this issue.

The issue is caused by the workspace dependencies being compiled before the `xtask`. currently the workspace dependencies are required for everything. 
My theory is that if we make every workspace dependency optional, it's possible to build the `xtask` crate without any issue. And I think it's reasonable because if you build a single crate, you should build only it's dependency and not the entire workspace

Reply:
Unfortunately, workspace dependencies cannot be optional. It literally triggers errors like:

```
❯ cargo xtask
error: failed to parse manifest at `/home/vadorovsky/src/pulsar/Cargo.toml`

Caused by:
  bpf-builder is optional, but workspace dependencies cannot be optional
```

However, I think that xtask wants LLVM installed not because of any workspace dependency, but because of this line

https://github.com/exein-io/pulsar/blob/vadorovsky/architest/xtask/src/cross.rs#L147

(it was there before)

But welp, I think I will just remove it. We can just run `cargo xtask cross --release` in CI and leave the option of debug builds in case someone would like to debug the binary in the VM.